plugins/pipedream/api_key.go

```diff
@@ -0,0 +1,102 @@
+package pipedream
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://pipedream.com/docs/api_key"),
+		ManagementURL: sdk.URL("https://console.pipedream.com/user/security/tokens"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.APIKey,
+				MarkdownDescription: "API Key used to authenticate to Pipedream.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 32,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+			{
+				Name:                fieldname.OrgID,
+				MarkdownDescription: "OrgId for the Pipedream organization.",
+				Secret:              true,
```

Comment:
The org ID isn't the authentication secret, and we usually do not conceal these by default.

Fields that aren't marked secret are still end-to-end encrypted in 1Password and only available to folks you've shared the item with. The difference is in how we handle them in the client, like showing the plaintext by default on screen.

```suggestion
				Secret:              false,
```